src/AirshipBindings.Android.Core/AirshipBindings.Android.Core.csproj

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.props" Condition="Exists('packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.props')" />
+  <Import Project="packages\Xamarin.AndroidX.Migration.1.0.0\build\monoandroid90\Xamarin.AndroidX.Migration.props" Condition="Exists('packages\Xamarin.AndroidX.Migration.1.0.0\build\monoandroid90\Xamarin.AndroidX.Migration.props')" />
```

Comment:
This is Xamarin's AndroidX Migration package. We are supposed to be able to remove it once we completely stop using the old Android support libraries.

Reply:
we have

Reply:
Unfortunately, Xamarin.Firebase has not. It still requires Android Support libraries. By including AndroidX.Migration, the build succeeds, but the Android Support libraries are not linked in. The migration library redirects Android Support calls to AndroidX.
- https://devblogs.microsoft.com/xamarin/androidx-for-xamarin/
- https://github.com/xamarin/XamarinAndroidXMigration
I cleaned the migration libraries out of the components that don't include Xamarin.Firebase.